test/Microsoft.Health.Fhir.Shared.Tests.Crucible/CrucibleDataSource.cs

```diff
@@ -55,11 +55,6 @@ public static async Task<string[]> GetSupportedIdsAsync(CrucibleClient client)
                     .GetTestFhirServer(_dataStore, null)
                     .GetTestFhirClient(ResourceFormat.Json);
 
-                if (testFhirClient?.SecurityEnabled == true)
```

Comment:
Some form of this still needs to happen.

Reply:
when would the client be null?